lib/vanagon/platform/rpm.rb

```diff
@@ -33,7 +33,8 @@ def generate_package(project) # rubocop:disable Metrics/AbcSize
       # @param workdir [String] working directory to stage the evaluated templates in
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
+      # @param project [String] project used for trigger scripts
+      def generate_packaging_artifacts(workdir, name, binding, project)
         erb_file(File.join(VANAGON_ROOT, "resources/rpm/project.spec.erb"), File.join(workdir, "#{name}.spec"), false, { :binding => binding })
```

Comment:
Did you mean for `project` here to be the `project` param? If so, should be `#{project}`. If not, what is the `project` param used for?

Reply:
This has always been `project.spec.erb`. It's the name of the template before it is parsed. It will be renamed to `#{name}.spec` after the template is parsed.

Reply:
My question about what the `project` param is used for still holds.

Reply:
It's only used for deb packaging but was added to the method signature in all the places since it gets overridden